.analysis_options

```diff
@@ -59,7 +59,6 @@ linter:
 
     # === style rules ===
     - always_declare_return_types
-    - always_put_control_body_on_new_line
```

Comment:
The Flutter style guide is just for the Flutter repository itself. Plugins (and Flutter apps) should follow the Dart style guide, but the app/plugin author is free to decide.

In general, it makes most sense to just use `dartfmt`, and not worry about formatting at all, ever. Having a separate style guide (like the Flutter repo style guide) is just silly, but of course app and plugin authors can use whatever style guides they want.

We, however, should set a good example, and have our plugins and app examples formatted using dartfmt. It might not seem so important for Flutter apps and plugins, since they'll only ever be used with Flutter, but if you create a package that's meant to be shared between different frameworks (Flutter, AngularDart, etc.), having different "style guides" becomes a problem.

Embracing dartfmt also means you can focus on the actual code, instead of worrying about the whitespace between it.